packages/legacy/src/react-to-webcomponent.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { describe, it, expect } from "vitest"
+import matchers from "@testing-library/jest-dom/matchers"
+
+// import r2wc from "./react-to-web-component"
+
+expect.extend(matchers)
+
+describe("core", () => {
```

Comment:
`decribe("legacy" ...)` (also suggests I left 'core' in place in react tests too

Reply:
I'll update